frontend/src/pages/chat/components/chat-sidebar/styles.module.scss

```diff
@@ -12,6 +13,12 @@
   border-radius: 24px;
 }
 
+.plus {
+  display: flex;
+  height: 33px;
+  margin-left: auto;
+}
```

Comment:
```suggestion
```

Reply:
When I remove height or max-height here I give the next result:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111736110/d22e6ef8-4b6a-4fe8-927c-dee64d395536)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111736110/798a3741-5269-40fd-8bce-b9e806969791)

Reply:
@Liza-Veis May I leave height or max-height property?

Reply:
no, because this mean that layout is incorrect, button itself should take this space and we should have the same styles for button in both places